Adding OrgSettings Docs (#21239)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/formssettings.md

---
title: "formsSettings resource type"

> description: "**Company wide settings for Microsoft Forms**"

Reviewer: ```suggestion
description: "Company-wide settings for Microsoft Forms"
```

author: "zadinsmo"
ms.localizationpriority: medium
ms.prod: "applications"
doc_type: resourcePageType
---

# formsSettings resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Company-wide settings for Microsoft Forms. 

Reviewer: Is there a parent resource for Microsoft Forms? If so, please add it here.

Author: No, this follows the same pattern of the other settings

> **Company wide settings for Microsoft Forms**

Reviewer: ```suggestion
Company-wide settings for Microsoft Forms. 
```

Reviewer: Please add the link to the parent entity for Microsoft Forms (if available).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |isBingImageSearchEnabled|Boolean|**Controls whether users can add images from Bing to forms.**|

Reviewer: ```suggestion
|isBingImageSearchEnabled|Boolean|Controls whether users can add images from Bing to forms.|
```

> |isExternalSendFormEnabled|Boolean|**Controls whether users can send a link to a form to an external user.**|

Reviewer: ```suggestion
|isExternalSendFormEnabled|Boolean|Controls whether users can send a link to a form to an external user.|
```

> |isExternalShareCollaborationEnabled|Boolean|**Controls whether users can collaborate on a form layout and structure with an external user.**|

Reviewer: ```suggestion
|isExternalShareCollaborationEnabled|Boolean|Controls whether users can collaborate on a form layout and structure with an external user.|
```

> |isExternalShareResultEnabled|Boolean|**Controls whether users can share form results with external users.**|

Reviewer: ```suggestion
|isExternalShareResultEnabled|Boolean|Controls whether users can share form results with external users.|
```

> |isExternalShareTemplateEnabled|Boolean|**Controls whether users can shareform templates with external users.**|

Reviewer: ```suggestion
|isExternalShareTemplateEnabled|Boolean|Controls whether users can share form templates with external users.|
```

> |isInOrgFormsPhishingScanEnabled|Boolean|**Controls whether phishing protection is run on forms created by users, blocking creation of forms if common phishing questions are detected.**|

Reviewer: ```suggestion
|isInOrgFormsPhishingScanEnabled|Boolean|Controls whether phishing protection is run on forms created by users, blocking the creation of forms if common phishing questions are detected.|
```

> |isRecordIdentityByDefaultEnabled|Boolean|**Controls whether names of users who fill out forms are recorded.**|

Reviewer: ```suggestion
|isRecordIdentityByDefaultEnabled|Boolean|Controls whether the names of users who fill out forms are recorded.|
```
